### DISCUSSION TITLE
Does the password on the SSH certificate expire.

### DISCUSSION THREAD

**Discussion by a4ginatl** [codefori/vscode-ibmi#1349] [General]
My VSC was working with an older machine and then stopped.
I suspect the SSH password I used expired or something happened after an upgrade.
How can I check the password or how do I replace it?
Thanks, Darryl.

**Comment by worksofliam** [comment]
You can right click on the connection, select Login Settings, and update the password or key there.

**Comment by a4ginatl** [comment]
Thank you
